kiva/fonttools/tests/test_scanner.py

```diff
@@ -0,0 +1,304 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+import os
+import sys
+import unittest
+from unittest import mock
+
+from fontTools.ttLib import TTFont
+from pkg_resources import resource_filename
+
+from .._scanner import (
+    _afm_font_property, _build_afm_entries, _ttf_font_property,
+    create_font_list, FontEntry, scan_system_fonts
+)
+
+data_dir = resource_filename("kiva.fonttools.tests", "data")
+is_macos = (sys.platform == "darwin")
+is_windows = (sys.platform in ("win32", "cygwin"))
+is_generic = not (is_macos or is_windows)
```

Comment:
im not entirely sure what's a generic platform - does generic here mean linux?

Reply:
Generic unix? Anything not macOS or Windows with X11, basically. That could be a lot of platforms, but in practice it's linux and freebsd (and solaris?).